### ISSUE TITLE
Week starts on the wrong day when Weekly view is opened from Event List widget

### ISSUE THREAD

**Issue by kubuntu-newbie** [SimpleMobileTools/Simple-Calendar#2162]
to duplicate:
configure Calendar to "Start week on Sunday" in General
configure Calendar to "View to open from the event list widget" as Weekly view in Widgets

when opening Calendar app from the home screen, Calendar will display week starting on Sunday 0 expected behavior
when opening Calendar by clicking on the Event List widget, Calendar will display week starting on Saturday, may be ignoring time zone?

**Comment by Aga-C** [comment]
Do you have `Start week with the current day` option enabled? And where exactly do you tap in the widget?

**Comment by naveensingh** [comment]
Hello!

Adding to @Aga-C's questions, what timezone are you in? Did this happen in previous versions too?

**Comment by kubuntu-newbie** [comment]
> Do you have `Start week with the current day` option enabled? And where exactly do you tap in the widget?

No, I don't want to have week displayed starting with the current day, I want to see the current week in my locale format, starting with Sunday.

I tap on the date/day of the week text in the top left corner of the events view widget, the week view opens as configured but as I described above, it shows it starting on Saturday, not Sunday. Expected behavior is to see the week from Sunday and ending on Saturday.

I'm in the MST time zone (-7 GMT)

**Comment by priestdo** [comment]
Running version 6.23.1 Pro, Android 12 on a Pixel 3, PDT. I also have this issue. Happy to provide any specifics you would find helpful.

**Comment by kubuntu-newbie** [comment]
Still broken...

**Comment by Batcastle** [comment]
This is not specific to the widget. This occurs even when opening weekly view from within the app.

**Comment by kubuntu-newbie** [comment]
still a problem in version 6.23.1 Pro
the fix seems to be changing to the local time before figuring out the starting day of the week

**Comment by JimmysMaths** [comment]
@priestdo SimpleMobileTools apps were sold to ad and spyware company. 

Fork is here:
https://github.com/FossifyOrg

Discussion Here:
https://github.com/SimpleMobileTools/General-Discussion/issues/241
